frontend/src/util/stores.ts

```diff
@@ -4,6 +4,21 @@ import { StateCreator, create } from "zustand";
 import IParticipant from "@/types/Participant";
 import ISession from "@/types/Session";
 import ITheme from "@/types/Theme";
+import IBlock from "@/types/Block";
+
+interface BlockSlice {
+    block?: IBlock;
+    blockLoading: boolean;
+    setBlock: (block: IBlock) => void;
+    setBlockLoading: (blockLoading: boolean) => void;
+}
+
+const createBlockSlice: StateCreator<BlockSlice> = (set) => ({
+    block: undefined,
+    blockLoading: true,
```

Comment:
Why is it necessary to add `blockLoading` to the store? Is this not sufficient to maintain the state of whether or not this is loading in the component which triggers the request to the block data from the backend?

Reply:
True, it's not even used rn